cli/tc-build.go

```diff
@@ -88,7 +99,18 @@ func (f FlagData) BuildResultsCmd(buildID int) error {
 func (f FlagData) BuildResultsForPRCmd(pr int) error {
 	tc := f.NewServer()
 
-	builds, err := tc.GetBuildsForPR(f.TC.Build.TypeID, pr, f.TC.Build.Latest, f.TC.Build.Wait, f.TC.Build.RunTimeout, f.TC.Build.RunTimeout)
+	prTests, err := f.GetPrTests(pr)
+	if err != nil {
+		return fmt.Errorf("error looking for tests for PR #%d: %w", pr, err)
+	}
+	var buildTypeID string
+	for s := range *prTests {
+		buildTypeID = viper.GetString("buildtypeid")
+		if s != "" {
+			buildTypeID += "_" + strings.ToUpper(s)
+		}
+	}
+	builds, err := tc.GetBuildsForPR(buildTypeID, pr, f.TC.Build.Latest, f.TC.Build.Wait, f.TC.Build.RunTimeout, f.TC.Build.RunTimeout)
```

Comment:
fix the white space here
```suggestion
	prTests, err := f.GetPrTests(pr)
	if err != nil {
		return fmt.Errorf("error looking for tests for PR #%d: %w", pr, err)
	}
	
	var buildTypeID string
	for s := range *prTests {
		buildTypeID = viper.GetString("buildtypeid")
		if s != "" {
			buildTypeID += "_" + strings.ToUpper(s)
		}
	}
	
	builds, err := tc.GetBuildsForPR(buildTypeID, pr, f.TC.Build.Latest, f.TC.Build.Wait, f.TC.Build.RunTimeout, f.TC.Build.RunTimeout)
```